react-native/react-native.d.ts

```diff
@@ -3985,7 +4124,7 @@ declare namespace  __React {
          * A RefreshControl component, used to provide pull-to-refresh
          * functionality for the ScrollView.
          */
-        refreshControl?: RefreshControl
+        refreshControl?: any
```

Comment:
This actually expects *an instance* of RefreshControl, i.e. `<RefreshControl />`, not the component itself.
Not sure how to say that.